fastlane/lanes/build.rb

```diff
@@ -223,7 +223,7 @@
     )
 
     upload_prototype_build(product: 'WordPress', version_name: version_name)
-    upload_gutenberg_sourcemaps(app: 'Wordpress', flavor: PROTOTYPE_BUILD_FLAVOR, build_type: PROTOTYPE_BUILD_TYPE, release_version: version_name)
+    upload_gutenberg_sourcemaps(app: 'Wordpress', flavor: PROTOTYPE_BUILD_FLAVOR, build_type: PROTOTYPE_BUILD_TYPE, release_version: version_name, build_version: current_build_code)
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-Android/blob/ab4fc3eaaf0c9e445c68ee09546eb4d602705c17/fastlane/lanes/build.rb#L226"></span>Layout/LineLength: Line is too long. [181/180]</td>
    </tr>
  </tbody>
</table>